client/src/components/layout/Alert.js

```diff
@@ -3,6 +3,8 @@ import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import { Alert as AntAlert } from 'antd';
 
+// TODO replace the ant component here with tailwind styled stuff
```

Comment:
If we plan on doing this in the tailwind issue, we should leave a comment saying `Issue#59` instead of `TODO`.